components/twitter/twitter.app.js

```diff
@@ -1,5 +1,7 @@
 const axios = require('axios')
-const querystring = require("querystring")
+const _ = require('lodash')
```

Comment:
Lodash is quite large, so avoid using it if you don't need it.

You are just doing 

```
if (_.get(response, 'status', 'Error') === 200) {
```

which is basically synonymous with `if ((response && response.status) || 'Error') === 200)`
which has no point in having that default since it will not change how the bool evaluates...

```
if (response && response.status === 200)
```